### PR TITLE
jewel: legal: remove doc license ambiguity

### PR DESCRIPTION
**Pull request by pdvian**
http://tracker.ceph.com/issues/23411

### CODE REVIEW

File: ceph.spec.in [hunk 1]
```diff
@@ -60,7 +60,7 @@ Version:	@VERSION@
 Release:	@RPM_RELEASE@%{?dist}
 Epoch:		1
 Summary:	User space components of the Ceph file system
-License:	LGPL-2.1 and CC-BY-SA-1.0 and GPL-2.0 and BSL-1.0 and GPL-2.0-with-autoconf-exception and BSD-3-Clause and MIT
+License:	LGPL-2.1 and CC-BY-SA-3.0 and GPL-2.0 and BSL-1.0 and BSD-3-Clause and MIT
```
Comment (smithfarm) [inline ceph.spec.in]: do not remove " and GPL-2.0-with-autoconf-exception" here - that is needed in jewel

File: doc/conf.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 import os
 
 project = u'Ceph'
-copyright = u'2010-2014, Inktank Storage, Inc. and contributors. Licensed under Creative Commons BY-SA'
+copyright = u'2016, Red Hat, Inc, and contributors. Licensed under Creative Commons Attribution Share Alike 3.0 (CC-BY-SA-3.0)'
```
Comment (smithfarm) [inline doc/conf.py]: change the license only, not the copyright statement

**Comment by pdvian** [comment]
@smithfarm made changes as requested. Do we also needs to retain origin License in other files ?

**Comment by smithfarm** [comment]
@pdvian Not sure what you mean. When resolving cherry-pick conflicts, the general idea is to refrain from introducing unrelated changes. Now when I compare the original master commit https://github.com/ceph/ceph/commit/e294dd59100d962753f4dee8f7156be5a7fba875 with the cherry-pick in this PR, I don't see unrelated changes, so it looks good to me now.

**Pull request merged**